frontend/src/components/opportunity/OpportunityHistory.tsx

```diff
@@ -48,6 +48,22 @@ const OpportunityHistory = ({ summary }: Props) => {
         title={t("archive_date")}
         content={formatHistoryDate(summary.archive_date)}
       />
+      <OpportunityHistoryItem
```

Comment:
Maybe more of a question for our designers - for these fields, they can only ever be set of the opportunity_summary.is_forecast = True due to how the data from the upstream system we pull this data in works - do we want these fields to only display in that case?